sdk/nodejs/cmd/dynamic-provider/index.ts

```diff
@@ -79,247 +100,318 @@ function getProvider(props: any): dynamic.ResourceProvider {
 // This allows the creation of the replacement resource to use the new provider while the deletion of the old
 // resource uses the provider with which it was created.
 
-function cancelRPC(call: any, callback: any): void {
-    callback(undefined, new emptyproto.Empty());
-}
+class ResourceProviderService implements provrpc.IResourceProviderServer {
+    [method: string]: grpc.UntypedHandleCall;
```

Comment:
This is to satisfy the type expected below in

```typescript
server.addService(provrpc.ResourceProviderService, resourceProvider);
```

Reply:
L o v e   i t